### ISSUE TITLE
`valid-v-on`: add more modifiers

### ISSUE THREAD

**Issue by usernamehw** [vuejs/eslint-plugin-vue#351]
* **ESLint Version:** 4.15.0
* **eslint-plugin-vue Version:** 4.2.0
* **Node Version:** 8.9.3

```javascript
"vue/valid-v-on": 1,
```

```vue
<template>
  <input @keyup.arrow-down="foo">
<!--        ^^^ [eslint-plugin-vue]
                [vue/valid-v-on]
                'v-on' directives don't support the modifier 'arrow-down'. 
-->
</template>
```
Since vue `^2.5.0` it's [possible](https://gist.github.com/yyx990803/9bdff05e5468a60ced06c29c39114c6b#v-on-automatic-key-modifiers) to use valid modifiers from  `KeyboardEvent.key`. 
It would be nice to expand the list of them at least for the most used ones.

**Comment by mysticatea** [comment]
Thank you for the report.

I confirmed it on [online demo](https://mysticatea.github.io/vue-eslint-demo/#eJxVTs0KgkAQfhWZsyZ4FIsi6AXsuJfFmWDJZpbZXc3Ed0+jDh2//2+GTpCghibSw/c20sGo4SxrHPsUs+OdpuR3VlXGAmXkvYGbiIGPrSn/UpCDpp4C1DMMicrB9g6LoRCGulpycIzEsXWvdbD6wevkt/3gbUdrAaGLol9y+3ZivLgn4Xn7mYO3GkhXjYJXIljeuKhDKw==)
